tests/test_tally.py

```diff
@@ -38,3 +39,9 @@ def test_parsing_tally_multiplier(self):
             print(test)
             input = Input([test], BlockType.DATA)
             data = parse_data(input)
+
+
+@pytest.mark.parametrize("line", ["fs14 -123", "fs12 -456 t"])
```

Comment:
Let's test:

```python
["fs14 -123",      # valid
 "fs12 -456 t",    # valid
 "fs11 -1 -2",     # valid
 "fs16 +1 +2 c",   # valid
 "fs17 -1 -2 t c", # valid
 "fs15  2"         # invalid: can't be used with f5
]
```

Reply:
I'm hesitant to enforce `fs15` because that's very much so a semantic test that I think should be covered when `TallySegment` is fully implemented so I think it's out of scope here and should be wrapped up into #332 .